src/cli/commands/run.js

```diff
@@ -26,7 +26,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const visitedBinFolders = new Set();
   let pkgCommands = [];
   for (const registry of Object.keys(registries)) {
-    const binFolder = path.join(config.cwd, config.registries[registry].folder, '.bin');
+    const binFolder = path.resolve(config.cwd, config.registries[registry].folder, '.bin');
```

Comment:
It looks like we use `path.join(config.cwd)` in several places in the code. Would it be worth updating https://github.com/yarnpkg/yarn/blob/master/src/config.js#L358 to be `path.resolve(opts.cwd)` for future issues related to relative cwd path?

Reply:
actually, that's a good idea... @BYK thoughts? If we change it in `config` then `config.cwd` would always be an absolute path. I can't think of any situation where that would be bad.

Reply:
+1 from me. I'm hoping tests will protect us from any unforeseen consequences.

Reply:
Pushed a new commit that adds a `path.resolve` to `config.cwd` to ensure it is always an absolute path.